Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM alpine:3.7
```

Comment:
@thaffenden you can use `https://github.com/vidsy/go-base` here instead which basically does everything below so you just need to then add the binary.